chromium_src/chrome/elevation_service/elevator.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_vpn/common/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+#include "brave/components/brave_vpn/common/wireguard/win/wireguard_utils_win.h"
+#include "brave/installer/win/util/brave_vpn_helper_utils.h"
+#endif
+
+#include "src/chrome/elevation_service/elevator.cc"
+
+namespace elevation_service {
+
+HRESULT Elevator::InstallVPNServices() {
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+  if (!brave_vpn::IsBraveVPNHelperServiceInstalled()) {
+    HRESULT hr = brave_vpn::InstallBraveVPNHelperServiceImpersonated();
```

Comment:
where is the corresponding uninstall for these?

Reply:
I see it now, but I think we should drop "Impersonated" here. I think it's just confusing things and leads to inconsistent naming between the IsInstalled, Install and Uninstall methods

Reply:
Impersonated is important to me for this case - because it lets the reader know that Windows impersonation is going to happen (which is not a regular thing). The `*Impersonated()` methods do the impersonation, the calls themselves, and then reverts back to user.

Reply:
but why is there a non impersonated version of the method? If we need admin, how can we be calling it directly without admin?